runtime/rococo/src/lib.rs

```diff
@@ -2043,3 +2063,38 @@ sp_api::impl_runtime_apis! {
 		}
 	}
 }
+
+#[cfg(all(test, feature = "try-runtime"))]
+mod remote_tests {
+	use super::*;
+	use frame_try_runtime::runtime_decl_for_TryRuntime::TryRuntime;
+	use remote_externalities::{
```

Comment:
I cannot find `remote-externalities` in the `Cargo.toml` of Rococo.  
So this should not compile. The CI is not checking that as it seems :man_facepalming:

Reply:
You can probably ignore this. Will fix it in a follow up.

Reply:
How exactly should this test be executed? `cargo test --features try-runtime` compiles just fine, but I agree it should fail.